docs/examples.md

```diff
@@ -80,6 +89,18 @@ Build and run WildFly applications on CentOS 7. For more information about using
 
 ## Binary example
 
+### java 
+
+Java can be used to deploy binary artifact
+
+```sh
+  git clone https://github.com/spring-projects/spring-petclinic.git
+  cd spring-petclinic
+  mvn package
+  odo create java test3 --binary target/*.jar
```

Comment:
Not sure you want to push **all** jars all the time. Usually, you only push the uber-jar…

Reply:
Generally speaking, when packaging as an uber jar, you will only get a single jar in target - the uber jar. If you had code the name of that, simply things like maven version changes can break automation

Reply:
Generally true, yes but still risky :)

Reply:
OK - how about `odo create java test3 --binary target/spring-petclinic-*.jar` ? I'm trying to avoid spring reving the version and breaking the example.